docs/source/concept_models/vivarium_us_cvd/concept_model.rst

```diff
@@ -211,20 +220,32 @@ Variations on the above intervention increased support time or added meal replac
 5.2.1 Population description
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-  - cohort type
-  - cohort length
-  - age and sex structure
-  - time step
-  - fertility
-  - stratifications 
+  - Cohort type: 
+  	- Prospective open cohort of adults aged 25 and above. New agents should age into the population at age_start
```

Comment:
We will probably implement this as a _closed_ cohort, starting at age 3.  Then the youngest simulants will be turning 25 when the sim ends. Then you can analyze the 25+ population as if it were an open cohort.

Reply:
starting at a range of 3-125
age range: 3 + duration of study - 125 + duration of study 
conscious of size of overall population vs size of initialized/observed group
doesn't matter that we don't have values for some inputs by age
modeling 3-25, but not observing; need to model correctly so that we end up with the right folks at 25